cspell.json

```diff
@@ -3,70 +3,10 @@
   "$schema": "https://raw.githubusercontent.com/streetsidesoftware/cspell/master/cspell.schema.json",
   "language": "en",
   "words": [
```

Comment:
Most of these changes are alphabetizing the `words` array (which VSCode decided to do automatically after I added a few new words to it). Hopes that's ok.